src/components/ui/Panel.jsx

```diff
@@ -202,6 +203,7 @@ class Panel extends React.Component {
             className={classnames('panel', size, className, {
               'panel--white': white,
               'panel--holding': holding,
+              'panel--pj': dataSource === 'pages_jaunes',
```

Comment:
Let's think about separation of concerns. `Panel` is a generic component which doesn't need to know about its content. So it shouldn't complexified with stuff specific to a single feature.
As it's already possible to pass a custom `className` to it, you have a proper alternative which doesn't require you to modify `Panel` code for that.

Reply:
Also, Panel has been refactored in master some time ago. We need to rebase this code on that.